### ISSUE TITLE
About how many epochs are needed for training?

### ISSUE THREAD

**Issue by mizumasa** [MasazI/cnn_depth_tensorflow#6]
Thank you for your code.
I am testing your code, but train loss doesn't decrease.
Perhaps my examination is wrong (I changed a few codes to use tensorflow v1.0).
About how many epoch does this training want?
In my examine, train loss goes around 1600. 〜 2500.  in first 0〜10 epochs.
Would you give me a advice?

In tensorflow v1.0 we can't use these module below.

tf.mul() -> multiply
tf. sub() -> subtract
tf.concat(3, [fine1_dropout, coarse7_output]) -> tf.concat([fine1_dropout, coarse7_output],3)
